CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### Changes since v0.13
 
 - Change `Lucky::BaseApp` to `Lucky::BaseAppServer`
+- `time_ago_in_words` returns a `String` instead of an `text` element (IO)
```

Comment:
0.14 has already been released, so this would more likely go under `### Changes since v0.14`. However, with that said, maybe all changelog changes can just go in a single commit? Kind of a pain to update all at once, but it helps to know what docs will need to be updated and such..